### PR TITLE
Run cmd

### PR DESCRIPTION
**Pull request by baude**

### CODE REVIEW

File: Atomic/objects/container.py [hunk 1]
```diff
@@ -100,3 +100,7 @@ def name(self, value):
     @property
     def user_command(self):
         return self._user_command
+
+    @user_command.setter
+    def user_command(self, value):
+        self._user_command = value
```
Comment (rhatdan) [inline Atomic/objects/container.py]: Add Return

**Comment by rhatdan** [comment]
Just a nit and LGTM

**Comment by baude** [comment]
updated per comments.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ ea5368e

**Pull request closed without merging**